tools/unicode_properties_parse/grapheme_break_property_data_gen.py

```diff
@@ -0,0 +1,95 @@
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception

```

Comment:
There's no copyright information in these Python programs - should these have Microsoft copyrights?